### PR TITLE
contact form section

### PR DESCRIPTION
**Pull request by brightbrain20**
my partner and I created a contact-me section which has a form to take in information.

- Name field accepts up to 30 chars.
- Email field accepts only emails.
- All fields have placeholder text.
- A user cannot submit the form if one or more of the fields is empty.
- Collect data by using the [Formspree service](https://formspree.io/html).
- we stick to the design as much as possible (e.g., font, colors, images, text, margins)

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
> Hi @brightbrain20,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, that doesn't include the input and commit of your partner, since this task is a pair programming task. Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -232,6 +232,53 @@ <h2>About me</h2>
...
+         so if you'd like to chat please get in touch.
+      </p>
+    </div>
+   <div>
+    <form action="https://formspree.io/f/mwkzggnv" method="post">
+      <input type="text" placeholder="Full name" maxlength="30" id="text" name="text" required>
+      <input type="email" id="email" name="email" placeholder="Email address" maxlength="30" required>
+      <textarea name="" id="" cols="10" rows="5" required maxlength="500" name="textarea" id="textarea"  placeholder="Enter text here"></textarea>
+      <button class="Multi-Post-project button form-button" type="submit">Get in touch</button>
+    </form>
```
Comment (wayungi) [inline index.html:249]: - [x] Please use flexbox to lay out elements in the form. This is a project requirement. 🤝 

ins

Comment (wayungi) [reply index.html:249]: - [x] Please implement the button states on hover and active as required in the project requirements 👍🏾 
btns

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @brightbrain20 , 
image

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Profesional README ✔️ 
- [ ] All linter checks passing ✔️ 
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ShadyShawkat) [inline index.html:249]: - Good job using `flex` for the form as the previous reviewer suggested. However, doing so resulted in changing the design. Kindly do the following:
   - [ ] Increasing the spacing around the form section to match the Figma template.
   - [ ] Centering the "Get in touch" button to match the given design.
   

| Your design | Figma design |
| --- | --- |
| image | image |

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @brightbrain20, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Image

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED 👍🎯  
Hi @brightbrain20, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ You fixed the spacing issue
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**